lib/_http_incoming.js

```diff
@@ -47,7 +47,7 @@ function IncomingMessage(socket) {
   // response (client) only
   this.statusCode = null;
   this.statusMessage = null;
-  this._client = socket; // deprecated
+  this.client = socket;
```

Comment:
Why don't we define `client` in `this` instead of `IncomingMessage.prototype` and issue the deprecation warning as it is in the old code?